xarray/core/dataarray.py

```diff
@@ -1097,10 +1097,28 @@ def fillna(self, value):
         if utils.is_dict_like(value):
             raise TypeError('cannot provide fill value as a dictionary with '
                             'fillna on a DataArray')
-        out = self._fillna(value)
+        out = ops.fillna(self, value, join=join)
         out.attrs = self.attrs
         return out
 
+    def combine_first(self, other):
```

Comment:
I think either we have `combine_first` for both Dataset and DataArray, or we lump everything into `fillna`.  The use of `combine_first` is just that there's this concrete direct analogy from `pandas`.  What's the decision here?